django/core/management/__init__.py

```diff
@@ -177,8 +177,14 @@ def fetch_command(self, subcommand):
         try:
             app_name = commands[subcommand]
         except KeyError:
-            # This might trigger ImproperlyConfigured (masked in get_commands)
-            settings.INSTALLED_APPS
+            if os.environ.get('DJANGO_SETTINGS_MODULE'):
+                # In case of the absence of `subcommand` due to misconfigured
```

Comment:
Yes, that's good. Suggested wording: If `subcommand` is missing due to misconfigured settings, the following line will retrigger an ImproperlyConfigured exception (get_commands() swallows the original one) so the user is informed about it.